test/parallel/test-dotenv.js

```diff
@@ -68,3 +68,10 @@ assert.strictEqual(process.env.TRIM_SPACE_FROM_UNQUOTED, 'some spaced out string
 assert.strictEqual(process.env.EMAIL, 'therealnerdybeast@example.tld');
 // Parses keys and values surrounded by spaces
 assert.strictEqual(process.env.SPACED_KEY, 'parsed');
+// Test multiple-line value
+assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'THIS\nIS\nA\nMULTILINE\nSTRING');
+assert.strictEqual(process.env.MULTI_SINGLE_QUOTED, 'THIS\nIS\nA\nMULTILINE\nSTRING');
+assert.strictEqual(process.env.MULTI_BACKTICKED, 'THIS\nIS\nA\n"MULTILINE\'S"\nSTRING');
+assert.strictEqual(process.env.MULTI_NOT_VALID_QUOTE, '"');
+assert.strictEqual(process.env.MULTI_NOT_VALID, 'THIS');
+assert.strictEqual(process.env.EXAMPLE, 'ignore export');
```

Comment:
According to [dotenv](https://github.com/motdotla/dotenv/blob/master/lib/main.js#L9), there may be an `export` keyword before the key. This keyword is ignored.
Unfortunately, I did not find a test for the `export` feature in the dotenv test suite.
This PR also adds support for the `export` keyword.